auto_tests/tox_test.c

```diff
@@ -94,7 +94,8 @@ static void print_status_m_change(Tox *tox, uint32_t friend_number, const uint8_
         return;
     }
 
-    if (length == sizeof("Installing Gentoo") && memcmp(message, "Installing Gentoo", sizeof("Installing Gentoo")) == 0) {
+    if (length == sizeof("Installing Gentoo") &&
```

Comment:
@iphydf can we force && on the 2nd line?

Makes it much easier to scan vertically and see what each if is actually checking?